src/main/webapp/WEB-INF/views/admin/oss/list-js.jsp

```diff
@@ -45,6 +45,15 @@
 			
 			$('#search').on('click',function(e){
 				e.preventDefault();
+
+                var searchOSSName = $("input[type=text][name=ossName]").val();
+                $("input[type=text][name=ossName]").val(searchOSSName.trim());
+
+                var searchLicenseName = $("input[type=text][name=licenseName]").val();
+                $("input[type=text][name=licenseName]").val(searchLicenseName.trim());
+
+                var searchHomepage = $("input[type=text][name=homepage]").val();
+                $("input[type=text][name=homepage]").val(searchHomepage.trim());
 				
```

Comment:
Indentation of 48~56 lines looks bad.

Reply:
> And I recommend to combine two commits into one commit and fix indentation.

That's a good idea. Rebase commits and use [squash](https://dev-yakuza.posstree.com/ko/git/git-squash/) option.

Reply:
OMG 😂 
Depending on whether the indentation is a space or a tab, it may not be recognized when uploading to GitHub.
Space are not recognized as indentation, so I changed space to tab. 😁
Thank you for reviewing the issues. 👍